src/runtime/qurt_hvx.cpp

```diff
@@ -2,6 +2,7 @@
 #include "HalideRuntimeQurt.h"
 #include "printer.h"
 #include "mini_qurt.h"
+#include "hexagon_remote/log.h"
```

Comment:
I actually swithced to log_printf after I tried using debug(user_context) and didn't see any output on mini-dm or logcat (w/ -debug set in the target)....either for the prefetch runtime calls or the other calls in this file that used debug(user_context).